docs/general/ddl/data-types.rst

```diff
@@ -2867,6 +2862,58 @@ constant array values.
 
 .. _data-types-geo:
 
+Nested arrays
+..............
+
+Nested arrays cannot be used directly in column definitions  (i.e. 
+``ARRAY(ARRAY(DOUBLE))`` is not accepted), but multiple arrays can be nested
+as long as objects come between them:
+
+::
+
+    CREATE TABLE SensorData (sensorID char(10),readings ARRAY(OBJECT AS (innerarray ARRAY(DOUBLE))));
+
+
+Nested arrays can still be used directly in input and output to UDFs:
+
+::
+
+    CREATE FUNCTION sort_nested_array("data" ARRAY(ARRAY(DOUBLE)),sort_dimension SMALLINT)
+    RETURNS ARRAY(ARRAY(DOUBLE))
+    LANGUAGE JAVASCRIPT
+    AS 'function sort_nested_array(data,sort_dimension) {			
+    		data = data.sort(function compareFn(a, b) {  
+    				if (a[sort_dimension]<b[sort_dimension]){return -1;}
```

Comment:
please fix a bit the indentation, it's too much to the right, comparing to the line above, it should be just +4 whitespaces.